server/views/challenges/showZiplineOrBasejump.jade

```diff
@@ -8,11 +8,10 @@ block content
                     | &nbsp;
                     i.ion-checkmark-circled.text-primary(title="Completed")
             hr
-            ol
+            ul
                 for step, index in description
                     .row.checklist-element(id="#{dashedName + index}")
-                        .col-xs-3.col-sm-1.col-md-2.padded-ionic-icon.text-center
-                            input(type='checkbox' class='challenge-list-checkbox')
+                        .col-xs-3.col-sm-1.col-md-2
```

Comment:
@atjonathan after removing that line the objectives shift towards left leaving space between video div and itself, it looks a little weird to me.